src/Fixer/LanguageConstruct/FunctionToConstantFixer.php

```diff
@@ -0,0 +1,189 @@
+                throw new InvalidFixerConfigurationException($this->getName(), sprintf('Configuration key must any of "%s".', implode('", "', self::$defaultConfiguration)));
+            }
+
+            $this->configuration[strtolower($key)] = self::$defaultConfiguration[$key];
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        for ($index = $tokens->count() - 4; $index > 0; --$index) {
+            $candidate = $this->isReplaceCandidate($tokens, $index);
+            if (false === $candidate) {
+                continue;
+            }
+
+            $this->fixFunctionCallToConstant(
+                $tokens,
+                $index,
+                $candidate[0], // brace open
+                $candidate[1], // brace close
+                $candidate[2]  // replacement
+            );
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Replace core functions calls returning constants with the constants.',
+            array(new CodeSample("<?php\necho phpversion();\necho pi();\necho php_sapi_name();")),
+            '',
+            'Configure any of `phpversion`, `php_sapi_name` and `pi`.',
+            array_keys(self::$defaultConfiguration),
+            'Risky when the any of the configured functions to replace are overridden.'
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should run before NativeFunctionCasingFixer
+        // must run before NoExtraConsecutiveBlankLinesFixer, NoSinglelineWhitespaceBeforeSemicolonsFixer, NoTrailingWhitespaceFixer and NoWhitespaceInBlankLineFixer
+        // must run after NoSpacesAfterFunctionNameFixer and NoSpacesInsideParenthesisFixer
+        return 1;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_STRING);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRisky()
+    {
+        return true;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param int    $braceOpenIndex
+     * @param int    $braceCloseIndex
+     * @param string $replacementConst
+     */
+    private function fixFunctionCallToConstant(Tokens $tokens, $index, $braceOpenIndex, $braceCloseIndex, $replacementConst)
+    {
+        $tokens->clearTokenAndMergeSurroundingWhitespace($braceCloseIndex);
+        $tokens->clearTokenAndMergeSurroundingWhitespace($braceOpenIndex);
+        $tokens[$index]->clear();
+        $tokens->insertAt($index, new Token(array(T_STRING, $replacementConst)));
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return array|false
+     */
+    private function isReplaceCandidate(Tokens $tokens, $index)
```

Comment:
`isFoo` should return boolean value, not data.
please convert it to `getReplaceCandidate : ?array`